client/extensions/wp-super-cache/state/settings/test/reducer.js

```diff
@@ -206,6 +209,81 @@ describe( 'reducer', () => {
 		} );
 	} );
 
+	describe( 'restoring()', () => {
+		const previousState = deepFreeze( {
+			restoring: {
+				[ primarySiteId ]: true,
+			}
+		} );
+
+		it( 'should default to an empty object', () => {
+			const state = reducer( undefined, {} );
+
+			expect( state.restoring ).to.eql( {} );
```

Comment:
What about `exporting` the `restoring` reducer so we can directly import it here (as opposed to relying on the combined reducer's `restoring` attr, which isn't quite the unit we want to test here)?